frontend/src/views/Stores.js

```diff
@@ -5,21 +5,24 @@ import Map from "../components/Map/Map";
 import { matchPath, Route } from 'react-router-dom';
 import Store from './Store';
 import StoreList from '../components/StoreList/StoreList';
+import { StoreListProvider } from 'context/StoreListContext';
```

Comment:
I am wondering if we need the context here. We could as well just put the logic in the component below and pass it from `Map` via a callback through a state in `Stores` to the StoreList. But if you'd rather do it this way thats fine.